pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/examples/ExampleViewModel.kt

```diff
@@ -29,27 +29,47 @@ class ExampleViewModel(application: Application) : AndroidViewModel(application)
      * Contents to display
      */
     val contents: StateFlow<List<Playlist>> = flow {
-        val listDrmContent = repository.getLatestMediaByShowUrn(SHOW_URN, 2).getOrDefault(emptyList())
-        val listTokenProtectedContent = repository.getTvLiveCenter(Bu.RTS, 2).getOrDefault(emptyList())
-        val playlist = Playlist(
-            title = PROTECTED_CONTENT_TITLE,
-            items = (listDrmContent + listTokenProtectedContent).map { DemoItem(title = it.title, description = it.lead, uri = it.urn) }
-        )
-        emit(LIST_STATIC_PLAYLIST + playlist)
-    }.stateIn(viewModelScope, started = SharingStarted.Eagerly, LIST_STATIC_PLAYLIST)
+        val listDrmContent = repository.getLatestMediaByShowUrn(SHOW_URN, PROTECTED_CONTENT_PAGE_SIZE).getOrDefault(emptyList())
+            .map { item ->
+                DemoItem(
+                    title = if (item.show?.title.isNullOrBlank()) {
+                        item.title
+                    } else {
+                        "${item.show?.title} (${item.title})"
+                    },
```

Comment:
Isn't there some Kotlin syntax to unwrap the show optional here? In Swift we would write something like:

```swift
let title = if let showTitle = item.show?.title, !showTitle.isEmpty {
    "\(showTitle) (\(item.title))"
}
else {
    item.title
}
```

which unwraps the optional automatically so that it is safe to use as non-optional in the corresponding code path.

Reply:
We don't have that ability in Kotlin 😞 
But I've updated that code to use a variable, to make it more readable in https://github.com/SRGSSR/pillarbox-android/pull/322/commits/1ab14839d5032f985daab4e8f2702a22b55cd719